charts/backstage/values.yaml

```diff
@@ -1,5 +1,12 @@
 nameOverride: developer-hub
+
 global:
+  # -- Catalog index configuration for automatic plugin discovery.
+  # The `install-dynamic-plugins.py` script pulls this image if the `CATALOG_INDEX_IMAGE` environment variable is set.
+  # The `dynamic-plugins.default.yaml` file will be extracted and written to `dynamic-plugins-root` volume mount.
+  pluginCatalogIndex:
```

Comment:
I'd suggest moving this to the root level, as it doesn't make sense to me to be in the global scope. Also we could be consistent with how the other Backstage and PostgreSQL image fields are defined (well with some exceptions):
```
pluginCatalogIndex:
  image:
    registry: quay.io
    repository: rhdh/plugin-catalog-index
    tag: "1.9"
```

Reply:
+1 to consistent approach, so that the values_patch.yaml downstream will work for this new operand image.

Aside: should you want to have digest pinned values here, this is how it's done downstream: https://gitlab.cee.redhat.com/rhidp/rhdh/-/merge_requests/403/diffs#67019b3fcc47da865d948af59dbb650ffd47327c